packages/cosmic-swingset/src/sim-params.js

```diff
@@ -139,4 +144,8 @@ export const DEFAULT_SIM_SWINGSET_PARAMS = {
   power_flag_fees: defaultPowerFlagFees,
   queue_max: defaultQueueMax,
   vat_cleanup_budget: defaultVatCleanupBudget,
+  installation_deadline_blocks: `${defaultInstallationDeadlineBlocks}`,
+  installation_deadline_seconds: `${defaultInstallationDeadlineSeconds}`,
+  bundle_uncompressed_size_limit_bytes: `${10 * 1024 * 1024}`,
+  chunk_size_limit_bytes: `${512 * 1024}`,
```

Comment:
For clarity, and the `chunk_size_limit_bytes` I recommended in the Go code:


```suggestion
  bundle_uncompressed_size_limit_bytes: `${10_000_000}`,
  chunk_size_limit_bytes: `${490_000}`,
```